collect.py

```diff
@@ -161,18 +183,19 @@ def run_test(bug, browser, driver, op_sequence=None):
         max_iter = 7 if op_sequence is None else len(op_sequence)
         for i in range(0, max_iter):
             if op_sequence is None:
-                elem_id = do_something(driver)
-                if elem_id is None:
+                elem_attributes = do_something(driver)
+                if elem_attributes is None:
                     print('Can\'t find any element to interact with on %s for bug %d' % (bug['url'], bug['id']))
                     break
-                saved_sequence.append(elem_id)
+                saved_sequence.append(elem_attributes)
             else:
-                elem_id = op_sequence[i]
-                do_something(driver, elem_id)
+                elem_attributes = op_sequence[i]
+                do_something(driver, elem_attributes)
 
-            print('  - Using %s' % elem_id)
+            print('  - Using %s' % elem_attributes)
+            image_file = str(bug['id']) + '_' + str(i) + '_' + browser
+            screenshot(driver, 'data/%s.png' % (image_file))
```

Comment:
Since this is breaking compatibility with the old way of doing things, before merging this we will also need a script to convert the current data to the new format.

Reply:
Done.